src/Microsoft.DotNet.Build.Tasks/PackageFiles/PerfTesting.targets

```diff
@@ -22,13 +21,13 @@
   </PropertyGroup>
   
   <!-- Perf testing assembly attribute -->
-  <ItemGroup Condition="'$(RunPerfTestsForProject)' == 'true' and '$(TestDisabled)' != 'true'">
+  <ItemGroup Condition="'$(IncludePerformanceTests)' == 'true'">
     <AssemblyInfoUsings Include="using Microsoft.Xunit.Performance%3B" />
     <AssemblyInfoLines Include="[assembly:OptimizeForBenchmarks]" />
   </ItemGroup>
 
   <!-- Copy over the performance runners to the test directory-->
-  <Target Name="CopyXunitExecutionDesktop" BeforeTargets="RunTestsForProject" Condition="'$(RunPerfTestsForProject)' == 'true'">
+  <Target Name="CopyXunitExecutionDesktop" AfterTargets="RestorePackages" Condition="'$(IncludePerformanceTests)' == 'true'">
```

Comment:
This seems like a weird target to hook, but I guess it will get the job done. Isn't there some target like "CopyTestToTestDirectory" or something like that we could hook into? Also, does this target get executed if you have already restored the packages?